src/fluidsf/generate_sf_maps_2d.py

```diff
@@ -53,6 +53,44 @@ def generate_sf_maps_2d(  # noqa: C901, D417
         dict:
             Dictionary containing the requested structure functions and separation
             distances for the x- and y-directions.
+            The returned dictionary may contain the following keys, with some keys
```

Comment:
@BrodiePearson would you mind reviewing this newly added docstring to make sure the definitions for the terms are appropriate? Specifically for the last 4 entries since these are newly written. The rest are just copies of the docstring in `calculate_sf_maps_2d.py`.

Reply:
I reviewerd all off the generate and calculate functions and made some minor edits to homogenize terminology across docstrings (e.g. "advective scalar" and "advective velocity" structure functions, and removing some occurrences of "traditional" where more precise names were needed).